pg2kinesis/__main__.py

```diff
@@ -32,8 +32,11 @@
               help='Attempt to on start create a the slot.')
 @click.option('--recreate-slot', default=False, is_flag=True,
               help='Deletes the slot on start if it exists and then creates.')
+@click.option('--change-kind', default='all',
```

Comment:
https://www.postgresql.org/docs/10/sql-altersubscription.html

the terminology that postgres uses is `operations`, so maybe `--operations=audit`